backend/utils.py

```diff
@@ -208,17 +258,7 @@ async def decorated_func(*args, **kwargs):
     return decorated_func
 
 
-# No longer using this inject stuff. got rid of circular imports
-# But this was how it was used:
-#
-#     inject_to_avoid_circular_imports('get_concepts', get_concepts)
-#     inject_to_avoid_circular_imports('CON', CON)
-#
-# INJECTED_STUFF = {}
-# def inject_to_avoid_circular_imports(name, obj):
-#     INJECTED_STUFF[name] = obj
-
-
+# TODO: never got these working, I think. Would be good to have, though. Not a high priority, but @joeflack4, look at some time
```

Comment:
Does this apply to this following issue? If so, I'll add a note about it:
- #637